hazzy.py

```diff
@@ -120,6 +120,9 @@ def __init__(self):
 
         # Module init
 
+        self.float_touchpad = Touchpad("float")
+        self.int_touchpad = Touchpad("int")
```

Comment:
Oh, because now the popups are created at program start instead of as needed. The show method you added just shows the already existing popup. I'm getting it! But I assume the memory usage is insignificant compared to having to parse the xml file and recreate the popup every time it is needed. Thank you.